content/ros/patterns.md

```diff
@@ -1 +1,42 @@
 # ROS Patterns
+## Service / Client
+
+This is an introduction practice for service and client
+
+## Instructions:
+1. Pull the newest version of the foundation course repository
```

Comment:
At this step, it's not clear that you want them to have the repository inside a catkin workspace; that only becomes clear in step 7. Is this mentioned somewhere else? But regardless, it should also be mentioned here.